src/Base/Chord/Subs.hs

```diff
@@ -34,7 +34,7 @@ import Base.Class.Rooted
 
 import Base.Core.Note
 import Base.Core.Quality.CQuality as CQ
-import Base.Core.Quality.IQuality as IQ
+import Base.Core.Quality.IQuality as IQ hiding (major)
```

Comment:
This was necessary since I added true smart constructors for the `Quality` type in this module, that is, functions correspondent to each of the constructors for the type. For the time being, the 'real' constructors are still exposed, but I'll probably commit that fix before we merge this in.